src/socks-to-rtc/socks-to-rtc.spec.ts

```diff
@@ -100,6 +115,19 @@ describe('SOCKS server', function() {
     server.startInternal(mockTcpServer, mockPeerConnection).then(
         server.stop).then(onceServerStopped).then(done);
   });
+
+  it('stop works before the PeerConnection connects', (done) => {
+    (<any>mockTcpServer.onceListening).and.returnValue(Promise.resolve(mockEndpoint));
```

Comment:
This is fine but I wonder if it wouldn't be clearer to have both TCP server and peerconnection use `noopPromise` here?